interactors/src/main/java/com/thefuntasty/interactors/interactors/BaseObservabler.kt

```diff
@@ -0,0 +1,20 @@
+package com.thefuntasty.interactors.interactors
+
+import io.reactivex.Observable
+
+abstract class BaseObservabler<ARGS, T> : BaseInteractor() {
+
+    protected abstract fun prepare(args: ARGS): Observable<T>
+
+    fun create(args: ARGS): Observable<T> {
+        return prepare(args).applySchedulers()
+    }
+
+    private fun Observable<T>.applySchedulers(): Observable<T> {
+        return compose { resultObservable ->
+            resultObservable
+                .subscribeOn(workScheduler)
+                .observeOn(resultScheduler)
+        }
+    }
+}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/abd629572327cbabcfd5483dec148fd3bc6b5ac2/interactors/src/main/java/com/thefuntasty/interactors/interactors/BaseObservabler.kt#L20"></span>The file BaseObservabler.kt is not ending with a new line.</td>
    </tr>
  </tbody>
</table>